tables/create_specimen.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE specimen(
+     id INT,
+     specimen_type VARCHAR(50),
+     plant_id VARCHAR(50),
+     location VARCHAR(512),
```

Comment:
Let's break location into two different fields, Geo-coordinates and location name.